citar.el

```diff
@@ -599,9 +702,36 @@ If no function is found, the DEFAULT function is called."
                   (citar-file--normalize-paths
                    citar-bibliography)))
 
-(defun citar--get-value (field entry)
-  "Return the FIELD value for ENTRY."
-  (cdr (assoc-string field entry 'case-fold)))
+;; Data access functions
+
+(cl-defun citar-get-data-entries (&optional &key filter)
```

Comment:
I started this as a more general function, with additional variables. It probably makes sense to require the one argument.
```suggestion
(cl-defun citar-get-data-entries (filter)
```